app/lib/data/database.dart

```diff
@@ -75,6 +75,17 @@ class DbManager {
     };
   }
 
+  // delete account
+  Future deleteAccount(int id) async {
+    await openDb();
+
+    await _database.delete("accounts", where: 'id = ?', whereArgs: [id]);
+
+    return {
+      'msg': 'All accounts deleted',
```

Comment:
Shouldn't say ALL accounts deleted here since we are only removing a single one.